.gitignore

```diff
@@ -206,14 +471,40 @@ poetry.toml
 # LSP config files
 pyrightconfig.json
 
-# End of https://www.toptal.com/developers/gitignore/api/python,macos
+### VisualStudioCode ###
+.vscode/*
+!.vscode/settings.json
+!.vscode/tasks.json
+!.vscode/launch.json
+!.vscode/extensions.json
+!.vscode/*.code-snippets
+
+# Local History for Visual Studio Code
+.history/
+
+# Built Visual Studio Code Extensions
+*.vsix
+
+### VisualStudioCode Patch ###
+# Ignore all local history of files
+.history
+.ionide
+
+# End of https://www.toptal.com/developers/gitignore/api/linux,macos,python,node,visualstudiocode,intellij
 
 # ignore all vscode, this is not standard configuration in this place
 .vscode
 
 # installed at build time
-wombat.js
+src/warc2zim/statics/wombat.js
 
-# temporary directories using during development
+# temporary directories used during development
 output
-tmp
\ No newline at end of file
+tmp
+
+# rule files are generated by rules/generate_rules.py
+src/warc2zim/rules.py
+javascript/src/fuzzyRules.js
+
+# wombatSetup.js is generated with rollup
+src/warc2zim/statics/wombatSetup.js
```

Comment:
I like your arguments. I proposed to keep it as it is for now since this is how we've already done on most Python scrapers. I still think current approach is wrong and your arguments are correct, so I've opened https://github.com/openzim/_python-bootstrap/issues/41 so that we track the decision + document what needs to be.